doc/conf.py

```diff
@@ -1,157 +1,12 @@
-import os
-import sys
-import time
+# Obtain shared config values
+import os, sys
 
-# Configuration file for the Sphinx documentation builder.
+sys.path.append(os.path.abspath(".."))
+sys.path.append(os.path.abspath("../.."))
 
-# -- Project information
-project = "Nuitka the Python Compiler"
-copyright = "%s, Kay Hayen and Nuitka Contributors" % time.gmtime().tm_year
-author = "Kay Hayen"
-release = version = ""
-
-# For autodoc to work
-sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
-from update import importNuitka
-importNuitka()
-del sys.path[-1]
-
-# -- General configuration
-
-extensions = [
-    # Sphinx's own extensions
-    "sphinx.ext.autodoc",
-    "sphinx.ext.extlinks",
-    "sphinx.ext.intersphinx",
-    "sphinx.ext.mathjax",
-    "sphinx.ext.todo",
-#    "sphinx.ext.viewcode",
-    # External extensions
-    "sphinx_copybutton",
-    "sphinx_design",
-#    "sphinx_inline_tabs",
-    "sphinxcontrib.youtube",
-    "sphinx-favicon",
-    "sphinx_sitemap",
-    # Disabled, loads to slow and cannot be limited to needing pages.
-    # "sphinxcontrib.asciinema",
-    # Blog extension
-    "ablog",
-]
-
-intersphinx_mapping = {
-    "python": ("https://docs.python.org/3/", None),
-    "sphinx": ("https://www.sphinx-doc.org/en/master/", None),
-}
-intersphinx_disabled_domains = ["std"]
-
-templates_path = ["_templates"]
-
-# Options for ABlog
-
-# The "title" for the blog, used in active pages.  Default is ``'Blog'``.
-blog_title = "Nuitka Blog"
-
-# Base URL for the website, required for generating feeds.
-# e.g. blog_baseurl = "http://example.com/"
-blog_baseurl = "https://nuitka.net"
-blog_path = "blog"
-blog_feed_fulltext = True
-blog_feed_archives = True
-blog_feed_length = None
-
-# TODO: Optimized feeds one day?
-x_blog_feed_templates = {
-    # Use defaults, no templates
-    "atom": {},
-    # Create content text suitable posting to social media
-    "social": {
-        # Format tags as hashtags and append to the content
-        "content": "{{ title }}{% for tag in post.tags %}"
-        " #{{ tag.name|trim()|replace(' ', '') }}"
-        "{% endfor %}",
-    },
-}
+from shared_conf import *
```

Comment:
Ah, this is where you are going, very well, but never do star imports, even if it is tedious to list all the things, at some point, I will want to know which symbols these are, but for now it's OK. That also explains the moving of the images I guess.